pylib/gyp/generator/make.py

```diff
@@ -981,12 +986,20 @@ def WriteActions(
             # libraries, but until everything is made cross-compile safe, also use
             # target libraries.
             # TODO(piman): when everything is cross-compile safe, remove lib.target
-            self.WriteLn(
-                "cmd_%s = LD_LIBRARY_PATH=$(builddir)/lib.host:"
-                "$(builddir)/lib.target:$$LD_LIBRARY_PATH; "
-                "export LD_LIBRARY_PATH; "
-                "%s%s" % (name, cd_action, command)
-            )
+            if self.flavor == "zos":
```

Comment:
`LIBPATH` is also the correct environment variable to use on AIX so we could also (probably as a follow up) add `self.flavor == "aix"`.